core/templates/dev/head/components/state-editor/state-editor.directive.ts

```diff
@@ -70,19 +72,20 @@ angular.module('oppia').directive('stateEditor', [
         'StateCustomizationArgsService', 'StateEditorService',
         'StateHintsService', 'StateInteractionIdService',
         'StateSolicitAnswerDetailsService', 'StateSolutionService',
-        'INTERACTION_SPECS',
+        'INTERACTION_SPECS', 'LearnerAnswerDetailsDataService',
```

Comment:
Please keep imports grouped and sorted.

Reply:
Actually I made this for testing purpose, removed it now